src/runtime/auto-prompt-manager.js

```diff
@@ -129,19 +129,19 @@ export class AutoPromptManager {
           autoPromptType: params.autoPromptType,
           callback: params.displayForLockedContentFn,
         });
-      }, (autoPromptConfig?.clientDisplayTrigger.displayDelaySeconds || 0) * SECOND_IN_MILLIS);
+      }, (clientConfig.autoPromptConfig?.clientDisplayTrigger.displayDelaySeconds || 0) * SECOND_IN_MILLIS);
```

Comment:
Could `clientConfig` be `undefined`? If so, the chaining operator should be placed right after it
```suggestion
      }, (clientConfig?.autoPromptConfig.clientDisplayTrigger.displayDelaySeconds || 0) * SECOND_IN_MILLIS);
```